Sources/MapboxNavigation/NavigationMapView+BuildingHighlighting.swift

```diff
@@ -23,17 +23,20 @@ extension NavigationMapView {
                                    in3D extrudesBuildings: Bool = true,
                                    extrudeAll: Bool = false,
                                    completion: ((_ foundAllBuildings: Bool) -> Void)? = nil) {
-        var foundBuildingIds = Set<Int64>()
+        highlightBuildings = highlightBuildings.filter{ coordinates.contains($0.key) }
```

Comment:
I see. Right now the `extrudeAll` feature is till kept. Because the identifiers will only be used to filter the building when `extrudeAll` is `false`.